templates/web/base/maps/_map_instructions.html

```diff
@@ -0,0 +1,18 @@
+<div class="keyboard-intructions-wrapper basic">
+    <div>
+        <div class="keyboard-key-icon zoom-in">Zoom in</div>
+        <div class="keyboard-key-icon zoom-out">Zoom out</div>
+        <div class="keyboard-key-icon left">Left</div>
+        <div class="keyboard-key-icon right">Right</div>
+    </div>
+    <span>You can move around with your keyboard</span>
+</div>
+
+<div class="keyboard-intructions-wrapper advanced">
+    <div>
+        <div class="keyboard-key-icon space">Space bar</div>
+    </div>
+    <span>Place/Remove pin pressing the space bar</span>
```

Comment:
Will need rewording. Not sure whether it’s clearer to say that the space bar will “drop a pin” or “place a pin” or whatever, or whether it’s better to say it’ll “start a new report”, since that‘s what people are _actually_ wanting to do. Hmmmm 🤔 I will have a think…